tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -42,6 +42,15 @@ def set_autostop_from_on_cluster_via_cluster_keep_warm():
     rh.here.keep_warm()
 
 
+def torch_exists():
+    try:
+        import torch  # noqa
```

Comment:
I would also try creating an array here. Sometimes torch imports but breaks as soon as you try to do anything.